docs/source/markdown/podman-pod-create.1.md.in

```diff
@@ -143,6 +143,11 @@ but only by the pod itself.
 
 @@option replace
 
+@@option restart
+
+The restart policy is set at the pod level so all containers in the pod have the same restart policy configured.
```

Comment:
I do not understand this restriction. The restart policy of a pod may very well be different than the one of its individual containers.